src/components/home/vacations-card.tsx

```diff
@@ -143,6 +150,22 @@ const VacationsCard = () => {
     fetchVacationsRequests();
   }, []);
 
+  /**
+   * Initialize logged in person's data.
+   */
+  const getPersons = async () => {
+    if (!persons.length) {
+      setLoading(true);
+      if (loggedInPerson || config.person.id) setLoading(false);
+      const fetchedPersons = await personsApi.listPersons({ active: true });
+      setPersons(fetchedPersons);
+    }
+  };
+
+  useMemo(() => {
+    getPersons();
+  }, [persons]);
```

Comment:
This wasn't used here before and yet the component was still able to access persons from the atom. It doesn't seem like this is needed to me, can it be removed? (Seems to work just fine when it is).

Reply:
Removed